2025/ig-wai.html

```diff
@@ -107,7 +107,7 @@ <h1 id="title"><i class="todo">DRAFT</i> Web Accessibility Initiative (WAI) Inte
   </div>
   <div id="background" class="background">
     <h2>Motivation and Background</h2>
-    <p>WAI IG is a well-established  Interest Group with over 2,500 participants in the email lists. WAI IG contributes  to a comprehensive accessibility approach at W3C.</p>
+    <p><abbr title="Web Accessibility Initiative Interest Group">WAI IG</abbr> is a well-established  Interest Group with over 2,500 participants in the email lists. WAI IG contributes  to a comprehensive accessibility approach at W3C.</p>
```

Comment:
```suggestion
    <p><abbr title="Web Accessibility Initiative Interest Group">WAI IG</abbr> is a well-established  Interest Group with over 2,500 participants in the email lists. WAI IG contributes to a comprehensive accessibility approach at W3C.</p>
```
Remove an extra space.